pyproject.toml

```diff
@@ -217,7 +217,9 @@ filterwarnings = [
 DJANGO_SETTINGS_MODULE = "tests.settings"
 
 # Test modules must have be named this way.
-python_files = "test_*.py"
+python_files = ["test_*.py", "benchmark_*.py"]
+python_functions = ["test_*", "benchmark_*"]
+python_classes = ["Test*", "Benchmark*"]
```

Comment:
You can see that the benchmarks are being run correctly e.g. here https://app.circleci.com/pipelines/github/octoenergy/xocto/802/workflows/32934bd1-2131-43ef-a135-474ea164ee21/jobs/781/parallel-runs/0/steps/0-110?invite=true#step-110-3189_79